tests/controller/elementControllerTest.js

```diff
@@ -125,15 +131,21 @@ before(async function () {
 
     // Create 2 QRcodes
     idQR1 = await QR.create({
-        code: "QRcode1"
+        code: codeQR1
     }).then((doc) => {
         return doc._id;
     });
     idQR2 = await QR.create({
-        code: "QRcode2"
+        code: codeQR2
     }).then((doc) => {
         return doc._id;
     });
+    await QR.create({
+        code: codeQR3
+    });
+    idQR4 =await QR.create({
```

Comment:
Forgot .then((doc) => {return doc_id;} );

Reply:
Yes. Good job for seeing that one.

Reply:
Corrected in next commit